pkg/openstack/loadbalancer.go

```diff
@@ -431,6 +432,19 @@ func getSecurityGroupName(service *corev1.Service) string {
 	return securityGroupName
 }
 
+func getLoadBalancerIP(service *corev1.Service) string {
+	if annotations := service.Annotations; annotations != nil {
```

Comment:
we can skip this check. `if val, ok := annotations[ServiceAnnotationLoadBalancerFloatingIP]; ok && val != "" {` is enough.

Reply:
Yes, it is used to avoid panic if there is no annotation. adopt your suggestion, it can cover my concern. 
https://go.dev/play/p/RDerITo3dqp?v=goprev